### DISCUSSION TITLE
Clipboard share (Android) for search

### DISCUSSION THREAD

**Discussion by Yourez** [organicmaps/organicmaps#1269] [Ideas]
Hi! 

One of the fastest metod to search place in OM by coordinates may be sharing from the clipboard. For comparison, I select coorfinates somewhere (browser, messenger, so on) and need to copy them, find OM somewhere, run OM, press "search" icon, paste them to the field and press Enter. If you implement share to OM I simply need to select coordinates string, choose Share in the context menu and select the OM. It's all!

It may be more suitable if you have series of points to search in OM.

**Comment by biodranik** [comment]
Good idea for iOS too! You can create an issue for that.
